### ISSUE TITLE
bug: Quickstart not working with newer versions of k8s

### ISSUE THREAD

**Issue by mfreeman451** [knative-extensions/kn-plugin-quickstart#480]
```shell
kn quickstart minikube
```

Pods fail to start in the `knative-serving` namespace:

```
2024/01/06 05:19:36 Timed out attempting to get k8s version: kubernetes version "1.25.3" is not compatible, need at least "1.26.0-0" (this can be overridden with the env var "KUBERNETES_MIN_VERSION")
```

**Comment by psschwei** [comment]
Thanks for reporting -- Kubernetes version will be bumped up in #479

**Comment by anencore94** [comment]
I encountered the same issue, and I found #479 is merged to the master branch.
When is the version with #479 expected to be released?

**Comment by psschwei** [comment]
This was released in v1.13 on Jan 24.
